core/model/src/main/java/it/unibz/inf/ontop/model/term/functionsymbol/db/impl/AbstractOrNullFunctionSymbol.java

```diff
@@ -58,27 +58,6 @@ public boolean canBePostProcessed(ImmutableList<? extends ImmutableTerm> argumen
         return true;
     }
 
-    @Override
```

Comment:
Please keep most of the logic in the class. Only the creation of function symbol (with a different arity) should be delegated to the subclasses.